training/trillium/Llama3.1-70B-MaxText-with-Storage/README.md

```diff
@@ -0,0 +1,160 @@
+# Instructions for training Llama3.1-70B-MaxText on TPU trillium (v6e-256) with Google Cloud Storage (GCS)
+
+## GCS Bucket setup
+Create a bucket with a dataset for dataloading and a bucket to write checkpoints. To create a regional HNS bucket use the following command:
+```
+gcloud storage buckets create gs://${BUCKET_NAME} location=$ZONE  defaultstorageclass=Standard enablehierarchicalnamespace uniformbucketlevelaccess
+```
+Checkout this [link](https://github.com/AI-Hypercomputer/maxtext/blob/b93beba652db6b3f4e6c82dc48a83b03229f5d3a/getting_started/Data_Input_Pipeline.md#tfds-pipeline) for downloading the Allenai c4 dataset which is used in this recipe.
+
+## XPK setup
+1. Please follow this [link](https://github.com/AI-Hypercomputer/tpu-recipes/blob/main/training/trillium/XPK_README.md) to create your GKE cluster with XPK.
+2. A FUSE adapter lets you mount and access Cloud Storage buckets as local file systems, so applications can read and write objects in your bucket using standard file system semantics. It adds pv and pvc to the cluster.
+https://github.com/AIHypercomputer/xpk?tab=readmeovfile#storage
+```
+cd ~/xpk
+
+python3 xpk.py storage attach $USERdatasetbucket type=gcsfuse project=$PROJECT cluster=$CLUSTER zone=$ZONE mountpoint=/tmp/dataset readonly=false bucket=<bucket> size=64 automount=false manifest=$RECIPE_REPO/tpu-recipes/training/trillium/Llama3.1-70B-MaxText-with-Storage/dataset_pvc.yaml
```

Comment:
$USERdatasetbucket should be "dataset" or "dataset_bucket" if we can use underscores or dashes.